docs/src/bootstrap.md

```diff
@@ -562,8 +563,9 @@ as owner of database `app`.
 application database will be changed to the value of `password` in secret. 
 
 !!! Important
-    Application database configuration will be disabled if the new cluster is 
-    created as a replica cluster, with replica mode enabled. 
+    In case of a replica cluster with replica mode enabled, the operator
```

Comment:
How about:

For a replica cluster with replica mode enabled, the operator will not create any database or user in the PostgreSQL instance, as these will be recovered from the original cluster.

?

Reply:
Sounds good.